### PR TITLE
Add user documentation

### PR DESCRIPTION
**Pull request by abe33**

### CODE REVIEW

**Comment by jansorg** [comment]
@abe33 Thanks, this is looking good!
- [ ] The 3rd screenshot in the section https://github.com/kiteco/atom-plugin/blob/2ade4d3a9df20e6babf15fdd877cd765a85cc889/docs/README.md#status-bar displays a "Launch Kite Enterprise" button. I think that we shouldn't mention the enterprise version as long as it's not an official product.

**Comment by abe33** [comment]
@jansorg It is indeed an official product:
kite_-_the_smart_copilot_for_programmers

**Comment by jansorg** [comment]
@abe33 I see. I assumed that this is a teaser, but I'm not sure.
@tarakju Is "Kite enterprise" available to companies already? (Should "Kite enterprise" be displayed in the docs at https://github.com/kiteco/atom-plugin/blob/2ade4d3a9df20e6babf15fdd877cd765a85cc889/docs/README.md#status-bar?)

**Comment by tarakju** [comment]
@abe33 @jansorg Kite Enterprise is not currently available to companies, however its a product we'd like to let people know we are going to support in the future (which is why its on the webpage). It will be a while before that becomes available to people so its probably best to take out the button, or leave it in for debug mode.

**Comment by abe33** [comment]
@tarakju @jansorg I've updated the screenshot with only the sole classic Kite button, same in vscode user docs

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+|`kite:open-sidebar`|Opens Kite's sidebar panel.|
+|`kite:close-sidebar`|Closes Kite's sidebar panel.|
+|`kite:toggle-sidebar`|Toggle Kite's sidebar panel.|
+
+### Settings
+
+
+|Setting|Description|
+|---|---|
+|`Show Kite Tour On Startup`|Whether to display the Kite tour on startup.|
```
Comment (dhung09) [inline docs/README.md]: change "Whether to display" to "Whether or not to display"

File: docs/README.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+|`Enable Completions`|Show auto-completions from Kite as Atom suggestions.|
```
Comment (dhung09) [inline docs/README.md]: change this to: "Show auto-completions from Kite as you code"

File: docs/README.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+|`Enable Hover UI`|Show hover informations when placing the mouse over a symbol|
```
Comment (dhung09) [inline docs/README.md]: change "informations" to "information"

File: docs/README.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+|`Display Expand View As`|This setting defines how the Kite expanded view will be displayed in Atom.|
+|`Sidebar Position`|When `Display Expand View As` is set to `sidebar` this setting defines where to place the sidebar in the Atom UI.|
+|`Sidebar Width`|When `Display Expand View As` is set to `sidebar` this setting defines the width of the sidebar.|
+|`Use Dock For Sidebar`|When the dock API is available in your Atom version Kite will use it to display the sidebar panel.|
```
Comment (dhung09) [inline docs/README.md]: change "When the dock API" to "If the dock API"

File: docs/README.md [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+|`Open Sidebar On Startup`|When `Display Expand View As` is set to `sidebar` this setting will make the sidebar appears on startup.|
```
Comment (dhung09) [inline docs/README.md]: change "appears on startup" to "appear on startup"

File: docs/README.md [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+|`Max Visible Suggestions Along Signature`|When both signature and completions are displayed at the same time, this setting controls the maximum number of suggestions visible.|
```
Comment (dhung09) [inline docs/README.md]: change this to "This setting controls the maximum number of completions shown when a function signature is also shown"

File: docs/README.md [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+|`Active Search Position`|Controls where the active search collapsed panel is displayed.|
+|`Hide Documentation When Signature Is Available`|If both a signature and documentation is available at the same only the signature will be visible.|
```
Comment (dhung09) [inline docs/README.md]: change this to: "This settings controls whether or not documentation is also shown when a function signature is shown"

File: docs/README.md [hunk 8]
```diff
@@ -0,0 +1,128 @@
...
+|`Collapse Long Summaries`|Whether to collapse or not long summaries in the sidebar panel.|
```
Comment (dhung09) [inline docs/README.md]: change this to: "Whether or not to collapse long summaries in the sidebar panel."

**Comment by jansorg** [comment]
@abe33 I think it's best to move the user docs to the top level, as you suggested. Could you do this before/after merging into master?
Thanks!

**Comment by jansorg** [comment]
@abe33 Thanks! Could you merge this into master, please?

**Pull request merged**